Utils/azure-explorer-common/src/com/microsoft/tooling/msservices/serviceexplorer/azure/webapp/WebAppModule.java

```diff
@@ -80,10 +81,13 @@ public void run() {
                     WebAppUtils.WebAppDetails webAppDetails = (WebAppUtils.WebAppDetails) event.object;
                     switch (event.opsType) {
                         case ADD:
+                            final IWebApp newWebApp = AzureWebAppMvpModel.getInstance()
```

Comment:
Why shall we use AzureWebAppMvpModel, shall we use az.webapp(id)?

Reply:
Will update codes here after refactor app service library to leverage Account service for authentication